js/src/common/Application.js

```diff
@@ -331,19 +331,31 @@ export default class Application {
       }
 
       const isDebug = app.forum.attribute('debug');
+      // contains a formatted errors if possible, response must be an JSON API array of errors
+      // the details property is decoded to transform escaped characters such as '\n'
+      const formattedError = error.response && Array.isArray(error.response.errors) && error.response.errors.map(e => decodeURI(e.detail));
 
       error.alert = new Alert({
         type: 'error',
         children,
         controls: isDebug && [
-          <Button className="Button Button--link" onclick={this.showDebug.bind(this, error)}>Debug</Button>
+          <Button className="Button Button--link" onclick={this.showDebug.bind(this, error, formattedError)}>Debug</Button>
         ]
       });
 
       try {
         options.errorHandler(error);
       } catch (error) {
+        if (isDebug) {
+          console.group(`${error.options.method} ${error.options.url} ${error.xhr ? error.xhr.status : ''}`);
```

Comment:
Sorry, but we need to verify whether this works in IE11. :stuck_out_tongue: 

[MDN](https://developer.mozilla.org/en-US/docs/Web/API/Console/group#Browser_compatibility) makes me think it does, but this comment [on caniuse](https://caniuse.com/#feat=mdn-api_console) isn't specific enough (Does this affect all functions? If not, which?) so that I am still worried:

> Only functions when developer tools are opened. Otherwise, the 'console' object is undefined.

Reply:
I don't see any errors in console in IE if I don't have the developer tools open - granted, IE doesn't show anything from before you open them. But it didn't break anything, so 🤷‍♂.

The `console` object is definitely not undefined when loading the page though - even if the developer tools haven't been opened. Logged through Custom Header script:

![image](https://user-images.githubusercontent.com/6401250/70376896-17d61800-18dc-11ea-8fc8-ae42a9943874.png)